guides_and_docs/machine_learning.md

```diff
@@ -0,0 +1,12 @@
+New to Machine leanring hpc using:
```

Comment:
typo! 'learning'
New to machine learning using High Performance Computing (HPC):